data-pipeline/src/pipeline/main.py

```diff
@@ -248,33 +248,38 @@ def pre_process_bfr(set_type, year):
         raw_container, f"{set_type}/{year}/BFR_3Y_raw.csv", encoding="unicode-escape"
     )
 
-    bfr_sofa, bfr_3y, bfr, bfr_metrics = build_bfr_data(bfr_sofa, bfr_3y)
+    academies_y2 = prepare_data(
+        pd.read_parquet(
+            get_blob("pre-processed", f"{set_type}/{year-2}/academies.parquet")
```

Comment:
Looking at the example message sent earlier:

```json
{
  "jobId": "5eabc249-e0e7-412c-9471-ebd05a66d4c4",
  "type": "comparator-set",
  "runType": "default",
  "runId": "9dbc9556-184b-45e6-bf85-3b3688807a9c",
  "year": "2022",
```

…`year` is a `str` when passed in, so the `year-2` calculation will fail (unless I'm missing an `int()` call somewhere.

Reply:
Good shout, I've added in some `int()`'s to deal with this

Reply:
For the sake of clarity/simplicity, you can probably just add a:

```python
year = int(msg_payload["year"])
```

…line in `handle_msg()` and pass that `year` around, rather than passing `msg_payload["year"]` as we're currently doing.

Reply:
@PsypherPunk if it's helpful I can change the datatype on the message?

Reply:
TBF, that would probably be a more sensible change (and allow us to validate the incoming message a little more).

@Chris-Lorch, you can ignore me an assume it's an `int`, if that's the case.

Reply:
@Chris-Lorch / @PsypherPunk I'll make that change to the message

Reply:
I'll leave it as is for now then if the data type in the message is specified